src/xcode/ENA/ENA/Source/Services/Risk/Provider/Helper/CachedAppConfiguration.swift

```diff
@@ -58,7 +58,7 @@ final class CachedAppConfiguration {
 
 	/// The location of the default app configuration.
 	private var defaultAppConfigPath: URL {
-		guard let url = Bundle.main.url(forResource: "default_app_config_113", withExtension: "") else {
```

Comment:
Don't know where else to ask. How is the requirement handled that we need to force a download of the new Appconfig from v2?

Reply:
I went with a really simple approach: I changed the key under which the app config is saved in the Store from "appConfigMetadata" to "appConfigMetadataV2". That is initially empty. So a new download is forced.

Reply:
Ah okay nice idea.